Content.Server/Fluids/EntitySystems/SpraySystem.cs

```diff
@@ -138,7 +149,21 @@ private void OnAfterInteract(Entity<SprayComponent> entity, ref AfterInteractEve
             if (TryComp<PhysicsComponent>(args.User, out var body))
             {
                 if (_gravity.IsWeightless(args.User, body))
-                    _physics.ApplyLinearImpulse(args.User, -impulseDirection.Normalized() * entity.Comp.PushbackAmount, body: body);
+                {
+                    // push back the player
+                    _physics.ApplyLinearImpulse(args.User, -impulseDirection * entity.Comp.PushbackAmount, body: body);
```

Comment:
What is the reason for using impulseDirection over impulseDirection.Normalized()?

Reply:
`rotation.ToVec()` already returns a normalized vector, so no reason to normalize it again. Better for performance.